File: myonlinefm/src/main/java/com/sec/myonlinefm/xmlcheck/LocalInfo.java
package com.sec.myonlinefm.xmlcheck;

/**
 * Created by SRC-TJ-MM-BinYang on 2018/1/3.
 */

public class LocalInfo {
    private String postion;
    private int channel;
    private String name;
    private int tag;



    public void setpostion(String postion) {
        this.postion = postion;
    }
    public void setchannel(int channel) {
        this.channel = channel;
    }
    public void setstationname(String name) {
        this.name = name;
    }
    public void settag(int tag) {
        this.tag = tag;
    }

    public String getpostion() {
        return postion;
    }
    public int getchannel() {
        return channel;
    }
    public String getstationname() {
        return name;
    }
    public int gettag() {
        return tag;
    }
}

File: myonlinefm/src/main/java/com/sec/myonlinefm/xmlcheck/LocalInfo.kt
package com.sec.myonlinefm.xmlcheck

/**
 * Created by SRC-TJ-MM-BinYang on 2018/1/3.
 */
class LocalInfo {
    private var postion: String? = null
    private var channel = 0
    private var name: String? = null
    private var tag = 0
    fun setpostion(postion: String?) {
        this.postion = postion
    }

    fun setchannel(channel: Int) {
        this.channel = channel
    }

    fun setstationname(name: String?) {
        this.name = name
    }

    fun settag(tag: Int) {
        this.tag = tag
    }

    fun getpostion(): String? {
        return postion
    }

    fun getchannel(): Int {
        return channel
    }

    fun getstationname(): String? {
        return name
    }

    fun gettag(): Int {
        return tag
    }
}